src/tailwind4.js

```diff
@@ -35,7 +35,7 @@ export const tailwind4 = {
             descriptors: defaultSyntax.properties,
         },
         source: {
-            prelude: "<string>",
+            prelude: "<url> | inline( <string># ) | not inline( <string># )",
```

Comment:
Double-check: Can `url()` be used with `@source`? I couldn't find that in the docs.